File: client/src/entities/BusinessTrip/model/types/BusinessTrip.ts
import { Employee } from "@/entities/Employee";

export interface BusinessTrip {
	id: string;
	employeeId?: string;
	employee?: Employee;
	laikotarpis?: Date | null;
	alga?: number;
	dienpinigai?: number;
	bankas?: number;
	baudos?: number;
	likutis?: number;
}

File: Domain/Entities/BusinessTrip.cs
namespace Domain.Entities
{
    public class BusinessTrip
    {
        public Guid Id { get; set; }
        public Guid EmployeeId { get; set; }
        public Employee Employee { get; set; }
        public DateOnly Laikotarpis { get; set; }
        public decimal Alga { get; set; }
        public decimal Dienpinigai { get; set; }
        public decimal Bankas { get; set; }
        public decimal Baudos { get; set; }
        public decimal Likutis { get; set; }

    }
}
